pkg/pma/util.h

```diff
@@ -0,0 +1,88 @@
+/// @file
```

Comment:
A pma-local util header could make sense for functionality that is solely useful for pma, but I think defining operations as basic as min,max,roundup,rounddown, etc here is an antipattern. Most likely these should be defined in c3.h or some other header which can be sourced by all components of vere.